minio/go/src/cloud/benchflow/commons/minio/minio.go

```diff
@@ -8,33 +8,11 @@ import (
 	"strings"
 	"bytes"
 	"fmt"
+	"crypto/md5"
+	"encoding/hex"
 )
 
-func StoreOnMinio(fileName string, bucket string, key string) {
-	config := minio.Config{
-		AccessKeyID:     os.Getenv("MINIO_ACCESSKEYID"),
-		SecretAccessKey: os.Getenv("MINIO_SECRETACCESSKEY"),
-		Endpoint:        os.Getenv("MINIO_HOST"),
-		}
-		s3Client, err := minio.New(config)
-	    if err != nil {
-	        log.Fatalln(err)
-	    }  
-	    object, err := os.Open(fileName)
-		if err != nil {
-			log.Fatalln(err)
-		}
-		defer object.Close()
-		objectInfo, err := object.Stat()
-		if err != nil {
-			object.Close()
-			log.Fatalln(err)
-		}
-		err = s3Client.PutObject(bucket, key, "application/octet-stream", objectInfo.Size(), object)
-		if err != nil {
-			log.Fatalln(err)
-		}
-	}
+const numOfHashCharacters int = 4
 
 func callMinioClient(fileName string, minioHost string, minioKey string) {
```

Comment:
@VincenzoFerme Just in case the API stops working properly again, we should keep the minio client call as well